frontend/src/components/GoalForm/index.js

```diff
@@ -146,7 +146,7 @@ export default function GoalForm({
         setDatePickerKey(goal.endDate ? `DPK-${goal.endDate}` : '00');
         setIsRttapa(goal.isRttapa);
         initialRttapa.current = goal.isRttapa;
-        setSelectedGrants(formatGrantsFromApi([goal.grant]));
+        setSelectedGrants(formatGrantsFromApi(goal.grants ? goal.grants : [goal.grant]));
```

Comment:
I also have a PR for this change. Is there ever a case when we would have only grant and not grants?

Reply:
https://github.com/HHS/Head-Start-TTADP/pull/1266